env/build_noaacloud_gnu.env

```diff
@@ -0,0 +1,17 @@
+#environment for building on NOAA-Cloud
+module purge
+module use /contrib/Mark.Potts/spack/share/spack/modules/linux-centos7-haswell
+module use /contrib/Mark.Potts/spack/share/spack/modules/linux-amzn2-skylake_avx512
```

Comment:
Will these modules eventually make it to a user-agnostic location?

Reply:
Ultimately, yes. We are still working on getting the platforms set up for EPIC's purposes, though.